src/services/collectibles_metadata_sources/index.ts

```diff
@@ -5,7 +5,7 @@ import { Mocked } from './mocked';
 import { Opensea } from './opensea';
 
 const sources: { [key: string]: CollectibleMetadataSource } = {
-    opensea: new Opensea(),
+    opensea: new Opensea({ rps: 5 }),
```

Comment:
PP: maybe this rps property can be configurable

Reply:
Mmmm initially thought the same, but I don't see anyone wanting to customize a lot this option. Also  this opensea metadata source is an example implementation so making it fully featured is not a priority right now.